app/controller/CategoriesController.php

```diff
@@ -89,6 +93,45 @@ public function viewAction($categoryId, $slug = null, $offset = 0)
     }
 
     /**
+     * Add new category
+     */
+    public function createAction()
+    {
+        if ($this->session->get('identity-admin') !== 'Y') {
+            $this->flashSession->error('You must have adminisrator rights');
```

Comment:
We shouldn't to provide an opportunity to hackers to think that there's something there (they just need to get right). In a perfect world we should throw a 403 here. But practice shows that the best choice is 404. To prevent Timing, Brute-force, DDos and other attacks

So, could you please forward to the error controller? https://github.com/phalcon/forum/blob/master/app/controller/ErrorController.php#L76